From cefe813458fa737a97bd797889a6716e09df913c Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Wed, 16 Oct 2024 10:05:02 +0200
Subject: [PATCH 1329/1695] dts: stm32: enable RTC on stm32mp215f-dk

Sets RTC peripheral status to okay

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 498b56a1db1a85adc79288bacd03945109cfa10f dts: stm32: enable RTC on stm32mp215f-dk]
Change-Id: Ic5d4cb727d428ddf75aee3e8da107f7d9600bd79
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/411879
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 0d50d0b41..a29b3f4bf 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -315,6 +315,10 @@
 	};
 };
 
+&rtc {
+	status = "okay";
+};
+
 &saes {
 	status = "okay";
 };
-- 
2.47.2

